js/os.ts

```diff
@@ -169,6 +169,66 @@ export function readFileSync(filename: string): Uint8Array {
   return new Uint8Array(dataArray!);
 }
 
+function createEnv(_msg: fbs.EnvRes): { [index:string]: string } {
+    const env: { [index:string]: string } = {};
+
+    for (let i = 0; i < _msg.mapLength(); i++) {
+        const item = _msg.map(i)!;
+
+        env[item.key()!] = item.value()!;
+    }
+
+    return new Proxy(env, {
+        set(obj, prop: string, value: string | number) {
+            setEnvVar(prop, value.toString());
+            return Reflect.set(obj, prop, value);
+        }
+    });
+}
+
+function setEnvVar(key: string, value: string) {
```

Comment:
`function setEnv(key: string, value: string): void {`

Add return type and rename to `setEnv` (to be inline with syscall names)